docs/content/preview/deploy/manual-deployment/system-config.md

```diff
@@ -35,6 +38,33 @@ As of CentOS 8, `ntp` is no longer available and has been replaced by `chrony`.
 $ sudo yum install -y chrony
 ```
 
+### PHC (PTP Hardware Clock) {{<tags/feature/tp>}}
+
+Hardware clocks are more accurate.
+
+See https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/configure-ec2-ntp.html#connect-to-the-ptp-hardware-clock
```

Comment:
To check if your AWS instance supports PHC, see [AWS PTP Hardware clock](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/configure-ec2-ntp.html#connect-to-the-ptp-hardware-clock)

Reply:
Done